src/app/app.module.ts

```diff
@@ -1,36 +1,31 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
-import { AppRoutingModule } from './app-routing.module';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { HttpClientModule } from '@angular/common/http';
+import '@angular/localize';
```

Comment:
Could you Please remove this ? and     "@angular/localize/init", add this in your angular.json file in polyfills

Reply:
yes, removed it and added to the angular.json